tutorials/R/download-ais-data/download-NEON-AIS-data.Rmd

```diff
@@ -0,0 +1,307 @@
+description: Tutorial for downloading NEON AIS data using the neonUtilities package and then exploring and understanding the downloaded data including seperating out into collection locations using the HOR variable. 
+dateCreated: '2020-05-19'
+authors: Bobby Hensley, Megan Jones
+contributors: 
+estimatedTime: 
+packagesLibraries: neonUtilities, ggplot2
+topics: data-management, rep-sci
+languageTool: R, API
+code1: R/download-explore/download-NEON-AIS-data.R
+tutorialSeries:
+urlTitle: download-neon-ais-data
+---
+
+This tutorial covers downloading NEON aquatic instrument system (AIS) data, using the 
+neonUtilities R package, as well as basic instruction in beginning to 
+explore and work with the downloaded data, including guidance in 
+navigating data documentation and understanding how to seperate data using the 
+horizontal (HOR) variable. 
+
+To set up our mini analyis in which we learn to access and work with the NEON AIS 
+data, our goal is to create a seperate R object for data from the various sensor
+locations and to create a plot of a variable of interest to compare the data from 
+different sensors. 
+
+<div id="ds-objectives" markdown="1">
+
+## Objectives
+
+After completing this activity, you will be able to:
+
+* Download NEON AIS data using the neonUtilities package.
+* Understand downloaded data sets and load them into R for analyses.
+* Seperate data collected at different sensor locations using the HOR variable.
+
+## Things You’ll Need To Complete This Tutorial
+To complete this tutorial you will need the most current version of R and, 
+preferably, RStudio loaded on your computer.
+
+### Install R Packages
+
+* **neonUtilities**: Basic functions for accessing NEON data
+* **ggplot2**: Plotting functions
+
+Some of these packages are on CRAN and can be installed by 
+`install.packages()`.
+
+### Additional Resources
+
+* <a href="https://github.com/NEONScience/NEON-Utilities/neonUtilities" target="_blank">GitHub repository for neonUtilities</a>
+
+</div>
+
+## Download files and load directly to R: loadByProduct()
+
+The most popular function in `neonUtilities` is `loadByProduct()`. 
+This function downloads data from the NEON API, merges the site-by-month 
+files, and loads the resulting data tables into the R environment, 
+assigning each data type to the appropriate R class. This is a popular 
+choice because it ensures you're always working with the most up to date data, 
+and it ends with ready-to-use tables in R. However, if you use it in
+a workflow you run repeatedly, keep in mind it will re-download the 
+data every time.
+
+Before we get the NEON data, we need to install (if not already done) and load 
+the neonUtilities R package. 
+
+```{r set-up-env, eval=F}
+# Install neonUtilities package if you have not yet.
+install.packages("neonUtilities")
+install.packages("ggplot2")
+
+```
+
+```{r load-packages}
+# Set global option to NOT convert all character variables to factors
+options(stringsAsFactors=F)
```

Comment:
Amazingly, this is now the default option in R 4.0.0! I think we should keep it in our tutorials since lots of people are on earlier versions, but I'm VERY excited.

Reply:
Awesome!  Agree on keeping it.